schedule/migration/migration_offline_scc_x86_64_all.yaml

```diff
@@ -0,0 +1,32 @@
+name: migration_offline_x86_64.yaml
```

Comment:
Suggest to update name:
schedule/migration/migration_offline_scc_all_patterns_x86_64.yaml

Reply:
rename to `schedule/migration/migration_offline_scc_all_patterns_x86_64.yaml`